lib/puppet/provider/jail/pyiocage.rb

```diff
@@ -44,30 +59,38 @@ def self.prefetch(resources)
   end
 
   def self.instances
+    default_properties = get_jail_properties('default')
```

Comment:
it's the system global configuration, if you so want. this has been fixed recently
but something else was broken, and i haven't figured out yet what / how https://github.com/iocage/iocage/issues/288